product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx

```diff
@@ -20,12 +20,18 @@ psql -W "postgres://edb_admin@xxxxxxxxx.xxxxx.biganimal.io:5432/edb_admin?sslmod
 !!! Note
     Avoid storing data in the postgres system database.
 
-## Notes on the edb_admin role
+## Admin roles
 
-- You can grant the edb_admin role superuser privileges for a cluster. If you grant superuser privileges, you must take care to limit the number of connections used by superusers to avoid degraded service or compromising availability.
+### pg_ba_admin
+In order for us to effectively manage the cloud resources and ensure users are protected against security threats, BigAnimal provides a special administrative role, pg_ba_admin. The edbadmin user is a member of the pg_admin role. The pg_ba_admin role has privileges similar to superusers. The pg_ba_admin role shouldn't be be used for day-to-day application operations and access to the role controlled carefully. See [pg_ba_admin role](pg_ba_admin_user) for details.
 
-  !!! note
-      Superuser privileges allow you to make Postgres configuration changes using `ALTER SYSTEM` queries. We recommend you don't do this because it might lead to an unpredictable or unrecoverable state of the cluster. In addition, `ALTER SYSTEM` changes aren't replicated across the cluster.
+### superuser
+When using your own cloud account, you can grant the edb_admin role superuser privileges for a cluster. If you grant superuser privileges, you must take care to limit the number of connections used by superusers to avoid degraded service or compromising availability.
```

Comment:
As I understand it, both the role and the user have the same name so there would be times when we would be talking about the user and times we are talking about the role. I searched the docs for where we are referring to the user and in the context, it seems correct. Ditto for role. Might need a little more info from you to identify exactly what needs to be fixed.

Reply:
Users are just special roles in Postgres - I just think we should be consistent.

Reply:
@ba-edb I think we would need to standardize on user then. Do you agree?

Reply:
@ba-edb I've reached out to Nidhi, the writer on EPAS, and have a better understanding of the roles vs user relationship in Postgres. I'm planning to publish what I have today and then do a new PR fixing the inconsistency and send it to review first to Nidhi and then to you, if you care to give it a quick review, as a quick follow on. Thanks for all your help.

Reply:
@ba-edb We have raised a PR where we have standardized the user vs role terminology when talking about edb_admin. We decided it made sense to standardize on user. Here is the [PR](https://github.com/EnterpriseDB/docs/pull/4321) in case you want to review it.